airflow-pipeline/spark_dag.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from airflow import DAG
+
+from airflow.contrib.operators.emr_create_job_flow_operator import (
+    EmrCreateJobFlowOperator,
+)
```

Comment:
Why is it formatted this way? Below too.

Reply:
done them consistensy